x/lazyledgerapp/types/payformessage.go

```diff
@@ -166,7 +167,7 @@ func (msg *MsgWirePayForMessage) GetCommitmentSignBytes(k uint64) ([]byte, error
 // to create a new SignedTransactionDataPayForMessage
 func (msg *MsgWirePayForMessage) SignedTransactionDataPayForMessage(k uint64) (*SignedTransactionDataPayForMessage, error) {
 	// add padding to message if necessary
-	msg.Message = PadMessage(msg.Message)
+	msg.padMessage()
```

Comment:
Shouldn't this be obsolete if the `MsgWirePayForMessage` was constructed via `NewMsgWirePayForMessage`?

Reply:
yes it is redundant, and it reminded me of why I put it there in the first place. The last PR #21  uses `MsgWirePayForMessage`s in the tests but obviously doesn't use the new `NewMsgWirePayForMessage` function, so those tests need a quick refactor along with getting rid of this `msg.padMessage`

Reply:
resolved by https://github.com/lazyledger/lazyledger-app/pull/32/commits/de38b729aeab05edc214823e90e913d293df6e8c and https://github.com/lazyledger/lazyledger-app/pull/32/commits/e15b95b650937e2ca811dd945371ac88a28e2e53